perceval/backends/askbot.py

```diff
@@ -0,0 +1,534 @@
+        return container_info
+
+    @staticmethod
+    def parse_user_info(update_info):
+        """Parse the user information of a given HTML container.
+
+        The method parses all the available user information in the container.
+        If the class "user-info" exists, the method will get all the available
+        information in the container. If not, if a class "tip" exists, it will be
+        a wiki post with no user associated. Else, it can be an empty container.
+
+        :param bs_question: beautiful soup answer container element
+
+        :returns: an object with the parsed information
+        """
+        user_info = {}
+        if update_info.select("div.user-info"):
+            # Get all the <a> elements in the container. First <a> contains the user
+            # information, second one (if exists), the website of the user.
+            elements = update_info.select("div.user-info")[0].find_all("a")
+            href = elements[0].attrs["href"]
+            user_info['id'] = re.search('\d+', href).group(0)
+            user_info['username'] = elements[0].text
+            user_info['reputation'] = update_info.select('span.reputation-score')[0].text
+            user_info['badges'] = update_info.select("span.badges")[0].attrs["title"]
+            try:
+                elements[1]
+            except IndexError:
+                pass
+            else:
+                user_info['website'] = elements[1].attrs["href"]
+            if update_info.select("img.flag"):
+                flag = update_info.select("img.flag")[0].attrs["alt"]
+                user_info['country'] = re.sub("flag of ", "", flag)
+            return user_info
+        elif update_info.select("p.tip"):
+            user_info = "This post is a wiki"
+            return user_info
+        else:
+            return
+
+
+class Askbot(Backend):
+    """Askbot backend.
+
+    This class retrieves the questions posted in an Askbot site.
+    To initialize this class the URL must be provided. The `url`
+    will be set as the origin of the data.
+
+    :param url: Askbot site URL
+    :param origin: identifier of the repository; when `None` or an
+        empty string are given, it will be set to `url` value
+    """
+    version = '0.1.0'
+
+    def __init__(self, url, tag=None):
+        origin = url
+
+        super().__init__(origin, tag=tag)
+        self.url = url
+        self.client = AskbotClient(url)
+        self.ab_parser = AskbotParser()
+
+    @metadata
+    def fetch(self, from_date=DEFAULT_DATETIME):
+        """Fetch the questions/answers from the repository.
+
+        The method retrieves, from an Askbot site, the questions and answers
+        updated since the given date.
+
+        :param from_date: obtain questions/answers updated since this date
+
+        :returns: a generator of items
+        """
+
+        if not from_date:
+            from_date = DEFAULT_DATETIME
+
+        from_date = from_date.timestamp()
```

Comment:
You have to call first to `datetime_to_utc(from_date)` to convert the date to UTC.